modules/apps/digital-signature/digital-signature-test/src/testFunctional/macros/DigitalSignature.macro

```diff
@@ -1,5 +1,23 @@
 definition {
 
+	macro assertAlertMessage {
+		AssertElementPresent(
+			locator1 = "DigitalSignatureDM#DIGITAL_SIGNATURE_MODAL_ALERT",
+			modalAlert = ${alertMessage});
+	}
+
+	macro assertDigitalSignaturePage {
+		AssertElementPresent(
+			key_title = ${assertPage},
+			locator1 = "Header#H1_TITLE");
+	}
+
+	macro clickCheckboxDM {
+		Click.javaScriptClick(
+			key_fieldName = ${checkboxDM},
+			locator1 = "Checkbox#CHECKBOX_NAME");
```

Comment:
I think here you could change  the macro name to: `"selectDMFolder"` to make it clearer